### PR TITLE
Fix/remove animation

### PR DESCRIPTION
**Pull request by lbebber**
#### What is the purpose of this pull request?
Removes animation on LoginContent due to performance issues (for example, [clicking on "Sign in" here](https://alssports.myvtex.com/) triggers a transition that is choppy on my decent MacBook Pro, let alone slower machines).

[The relevant commit is this one](https://github.com/vtex-apps/login/commit/1d4424823515aea8f05aef50366af8b131845f5f), the rest are formatting and removing dependencies.

Working example here: https://lbebber--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/LoginContent.js [hunk 1]
```diff
@@ -364,31 +374,11 @@ class LoginContent extends Component {
...
-                    keys={
-                      this.shouldRenderForm && render ? ['children'] : []
-                    }
-                    from={{ opacity: 0, transform: 'translateX(50%)' }}
-                    enter={{ opacity: 1, transform: 'translateX(0%)' }}
-                    leave={{ display: 'none' }}
-                  >
-                    {this.shouldRenderForm && render ? [render] : []}
-                  </Transition>
+                  {this.shouldRenderForm && render ? render() : null}
```
Comment (kevinch) [inline react/LoginContent.js]: What is this `render` variable? Any chance its name could be more explanatory?

**Comment by jgfidelis** [comment]
I did a login and it worked.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
### Removed
```

Comment (lbebber) [reply CHANGELOG.md]: I think this is more of a change on how it works rather than a removal of a feature.

Comment (kevinch) [reply CHANGELOG.md]: I agree with Breno on this one. Id even use "Fixed" since it solved a performance issue.

Comment (lbebber) [reply CHANGELOG.md]: "Removed" should be reserved for removal of public facing APIs, components, etc. This was not a bug either, so it is not a "Fix". It's a change in behaviour of a component.

Comment (kevinch) [reply CHANGELOG.md]: Imo if the problem was performance issue and therefore limiting the UX of the app then it should be a "Fixed" section. 
But potatoes patataoes.

**Pull request merged**